website/www/site/content/en/documentation/dsls/sql/extensions/create-external-table.md

```diff
@@ -644,6 +644,151 @@ TYPE text
 LOCATION '/home/admin/orders'
 ```
 
+## DataGen
+
+The **DataGen** connector allows for creating tables based on in-memory data generation. This is useful for developing and testing queries locally without requiring access to external systems. The DataGen connector is built-in; no additional dependencies are required.
```

Comment:
Website change takes effect nightly. We should note that this is available for Beam 2.67.0+